packages/cli/commands/sandbox/sync.js

```diff
@@ -0,0 +1,292 @@
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+  addTestingOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logger } = require('@hubspot/cli-lib/logger');
+const Spinnies = require('spinnies');
+const { initiateSync } = require('@hubspot/cli-lib/sandboxes');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logErrorInstance } = require('@hubspot/cli-lib/errorHandlers');
+const { ENVIRONMENTS } = require('@hubspot/cli-lib/lib/constants');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { getAccountConfig, getConfig, getEnv } = require('@hubspot/cli-lib');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/cli-lib/lib/urls');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { uiLine } = require('../../lib/ui');
+const {
+  getAccountName,
+  getAvailableSyncTypes,
+  pollSyncTaskStatus,
+} = require('../../lib/sandboxes');
+const {
+  isMissingScopeError,
+} = require('@hubspot/cli-lib/errorHandlers/apiErrors');
+
+const i18nKey = 'cli.commands.sandbox.subcommands.sync';
+
+exports.command = 'sync';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  await loadAndValidateOptions(options);
+
+  const { force } = options; // For scripting purposes
+  const config = getConfig();
+  const accountId = getAccountId(options);
+  const accountConfig = getAccountConfig(accountId);
+  const spinnies = new Spinnies({
+    succeedColor: 'white',
+  });
+
+  trackCommandUsage('sandbox-sync', null, accountId);
+
+  if (
+    // Check if default account is a sandbox, otherwise exit
+    // sandboxAccountType is null for non-sandbox portals, and one of 'DEVELOPER' or 'STANDARD' for sandbox portals. Undefined is to handle older config entries.
+    accountConfig.sandboxAccountType === undefined ||
+    accountConfig.sandboxAccountType === null
+  ) {
+    trackCommandUsage('sandbox-sync', { successful: false }, accountId);
+
+    logger.error(i18n(`${i18nKey}.failure.notSandbox`));
+
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  // Verify parent account exists in the config
+  let parentAccountId;
+  for (const portal of config.portals) {
```

Comment:
I think you can use this [getAccountConfig](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/lib/config.js#L318) util here.

Reply:
Good call, had lifted this from elsewhere and forgot to update. Pushing a new version soon